library/apisix

```diff
@@ -0,0 +1,12 @@
+Maintainers: Apache APISIX Docker Maintainers <dev@apisx.apache.org> (@soulbird)
+GitRepo: https://github.com/apache/apisix-docker.git
+
+Tags: 2.15.0, 2.15.0-debian, latest
```

Comment:
2.15.0 doesn't exist

Reply:
This is the tag of the final target image, not the repository's tag.